test/parallel/test-socket-write-after-fin-error.js

```diff
@@ -16,18 +16,18 @@ let gotServerError = false;
 
 const server = net.createServer(function(sock) {
   sock.setEncoding('utf8');
-  sock.on('error', function(er) {
-    console.error(`${er.code}: ${er.message}`);
-    gotServerError = er;
-  });
+  sock.on('error', function() {});
```

Comment:
Since we are changing this:
```suggestion
  sock.on('error', () => {});
```
Why the empty handler though? Shouldn't it be `mustCallAtLeast()` or `mustNotCall()`?

Reply:
To facilitate backporting. This way it works in all supported release lines.